src/components/05_pages/Permissions/Permissions.js

```diff
@@ -1,17 +1,156 @@
-import React from 'react';
-import { css } from 'emotion';
+import React, { Component, Fragment } from 'react';
+import makeCancelable from 'makecancelable';
+import { Markup } from 'interweave';
 
-const styles = {
-  title: css`
-    text-decoration: underline;
-  `,
-};
+import Loading from '../../Helpers/Loading';
+import { Table, TableBody, TableHeaderSimple } from '../../UI';
+
+const Permissions = class Permissions extends Component {
+  state = {
+    loaded: false,
+  };
+  componentDidMount() {
+    this.cancelFetch = makeCancelable(
+      Promise.all([
+        fetch(
+          `${
+            process.env.REACT_APP_DRUPAL_BASE_URL
+          }/admin-api/permissions?_format=json`,
+        ).then(res => res.json()),
+        fetch(
+          `${
+            process.env.REACT_APP_DRUPAL_BASE_URL
+          }/jsonapi/user_role/user_role`,
```

Comment:
You probably want to add the `Accept: application/vnd.api+json` header. This will enable 404/403 errors in JSON format (if any). Otherwise they'll come back as HTML errors.

Reply:
👍